builds/e2e/e2e.yaml

```diff
@@ -191,6 +191,61 @@ jobs:
     timeoutInMinutes: 120
 
     steps:
+    - bash: |
+        set -e
+        # Install PowerShell dependencies
+        sudo apt-get update
+        sudo apt-get install -y liblttng-ust0 jq
+        # Get the PowerShell tarball
+        release_id=$(curl -sSH 'Accept: application/vnd.github.v3+json' https://api.github.com/repos/powershell/powershell/releases/latest | jq '.id')
+        url=$(curl -sSH 'Accept: application/vnd.github.v3+json' https://api.github.com/repos/powershell/powershell/releases/$release_id/assets | jq -r ".[] | select(.name | contains(\"linux-arm64.tar.gz\")) | .browser_download_url")
+        echo "$release_id"
+        echo "$url"
+        file=$(basename $url)
+        curl -sSL $url -o /tmp/$file
+        # Install PowerShell
+        sudo mkdir -p /opt/microsoft/powershell/7
+        sudo tar zxf /tmp/$file -C /opt/microsoft/powershell/7
+        sudo chmod +x /opt/microsoft/powershell/7/pwsh
+        sudo ln -s /opt/microsoft/powershell/7/pwsh /usr/bin/pwsh
+      displayName: Install pwsh
+
+    - bash: |
+        set -e
+        # Install .NET Core dependencies
+        sudo apt-get update
+        sudo DEBIAN_FRONTEND=noninteractive apt-get install -y \
+            libc6 libgcc1 libgssapi-krb5-2 libicu66 libssl1.1 libstdc++6 zlib1g
+        # Install .NET Core
+        curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --channel 6.0
+        sudo ln -s $HOME/.dotnet/dotnet /usr/bin/dotnet
+      displayName: Install .NET Core
```

Comment:
I'm sure it does, but at the time it was easier to copy-paste the code we use in our 1ES-managed image repo than to adapt a DevOps task. Again, this is all throwaway code anyway...it will be removed once 1ES fully supports 1ES-managed images for ARM.